etc/config/wasm.defaults.properties

```
@@ -0,0 +1,21 @@
+compilers=&wasmtime
+objdumper=llvm-objdump
+objdumperType=llvm
+binaryHideFuncRe=^.*::(:?native_to_wasm_trampoline|wasm_to_native_trampoline|array_to_wasm_trampoline).*$
+
+group.wasmtime.compilers=wasmtime2001
+group.wasmtime.compilerType=wasmtime
+group.wasmtime.options=-Ddebug-info=y -Ccache=n
+group.wasmtime.supportsBinary=true
+group.wasmtime.supportsBinaryObject=true
+group.wasmtime.versionFlag=--version
+group.wasmtime.isSemVer=true
```

By adding
```suggestion
group.wasmtime.isSemVer=true
group.wasmtime.baseName=Wasmtime
```

You can then skip specifying the `.name` entry in each compiler entry, as the final name will end up being composed as `<group.baseName> <compiler.semver>`


This also applies to the `amazon` file